server/src-lib/Hasura/RQL/Types/SchemaCache.hs

```diff
@@ -16,6 +16,7 @@ module Hasura.RQL.Types.SchemaCache
 
        , TableInfo(..)
        , tiName
+       , tiDescription
```

Comment:
does this PR require this symbol to be exported?

Reply:
Yes, the warning in https://github.com/hasura/graphql-engine/commit/99174cca9b99ebb4b7d55d02fa1dcda150a3d57c is due to it not being exported, which causes an unused identifier warning. (The definition can’t just be deleted because it’s generated by `makeLenses`, and some of the other lenses *are* used.)